main/bamboo/bamboo.go

```diff
@@ -188,63 +188,84 @@ func listenToZookeeper(conf configuration.Configuration, eventBus *event_bus.Eve
 }
 
 func listenToMarathonEventStream(conf *configuration.Configuration, sub api.EventSubscriptionAPI) {
+	ticker := time.NewTicker(1 * time.Second)
+	go listenToMarathonEventStreamLoop(conf, &sub, ticker.C)
+}
+
+type eventBusSink interface {
+	Notify(payload []byte)
+}
+
+func listenToMarathonEventStreamLoop(conf *configuration.Configuration, sink eventBusSink, ticker <-chan time.Time) {
+	for _ = range ticker {
+		for _, marathon := range conf.Marathon.Endpoints() {
```

Comment:
I might add a log line here to indicate which Marathon endpoint we're now listening to